.github/workflows/loc-auto-merge.yml

```diff
@@ -1,5 +1,8 @@
 name: 'Auto-merge Validation'
 
+permissions:
```

Comment:
also add a content permission actually or it's going to fail on the checkout 
```suggestion
permissions:
	contents: read
```

Reply:
Sure, thanks!